quantum_annealing/Dwave_StructuralImbalance/Dwave_StructuralImbalance.ipynb

```diff
@@ -11,8 +11,7 @@
    "cell_type": "markdown",
```

Comment:
<p>As in the previous example, this example uses...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>